Example/DetailViewController.swift

```diff
@@ -72,9 +72,12 @@ class DetailViewController: UITableViewController {
             let end = CACurrentMediaTime()
             self.elapsedTime = end - start
 
-            for (field, value) in response!.allHeaderFields {
-                self.headers["\(field)"] = "\(value)"
+            if let _ = response{
```

Comment:
This should be `if let response = response`, and that way you don't have to explicitly unwrap the optional below `response!`.